lib/features/exhibition_flow/screens/voucher_code_screen.dart

```diff
@@ -107,17 +107,16 @@ class _VoucherCodeScreenState extends State<VoucherCodeScreen> {
                   FloatingActionButtonLocation.centerFloat,
               floatingActionButton: authState is! LoadingState &&
```

Comment:
Because we use the floatingActionButton option, we can get these kind of situations:
![image](https://github.com/givtnl/givt-app-kids/assets/13018117/3541e85c-9888-43a5-a4f4-0284b917fafc)

Reply:
I was replacing the buttons in their original place on the screen, not improving the screens; 
This is indeed a similar situation to what we discussed today, but tbh this PR & stories were so hard to keep within scope, improving the screens where I was replacing buttons was not within story, I would have to analyze each screen for these types of bugs.
I can change this one but i don't think should not be a blocker for the pr

Reply:
No, It's not a blocker! But if it's easy I would do it maybe.. or we put it in the backlog :)

Reply:
It's improved ;)